tests/requirements.txt

```diff
@@ -0,0 +1,6 @@
+pytest~=8.3.4
```

Comment:
I doubt we really need these specific patchlevels. Let's keep "major.minor" only here.

Reply:
Removed patch version.